go/apps/api/integration/multi_node_usagelimiting/README.md

```
@@ -0,0 +1,125 @@
+4. **ClickHouse validation**: Verify analytics data
+5. **Database consistency**: Check eventual consistency
+6. **Cleanup**: Automatic test resource cleanup
+
+## Regenerating Tests
+
+To add new test scenarios:
+
+1. Edit `generate_tests/main.go`
+2. Add new `TestCase` entries to `realisticCombinations` or `extremeEdgeCases`
+3. Run the generator:
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)